HW03_Graphs.Rmd

```diff
@@ -0,0 +1,174 @@
+---
+title: "HW03_Graphs"
+author: "Ricardo Muñiz Trejo"
+date: "20/7/2020"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r load libraries and set format, echo = FALSE}
+library("readr")
+library("readxl")
+library("ggplot2")
+theme_set(theme_bw())
+```
+
+
+## Introduction
+
+For this assignment, I decided to recreate some of the graphs of my master's thesis. Originally, my graphs were created with Origin 7 (images included for comparison). Because this is unpublished data, I censored the names, but basically the data presented here is from the physical-chemical characterization a couple of thermophilic enzymes.
```

Comment:
I appreciate that you included the "original" graphs as a reference! Made it very easy to make sure I was seeing the right data!